src/main/java/br/com/people/api/controller/PersonController.java

```diff
@@ -36,21 +37,22 @@
     @DeleteMapping(value = "/{id}")
     public ResponseEntity<Person> deleteById(@PathVariable long id) {
         log.info("The id {} was successfully deleted", id);
-        var person = dataBase.find(id);
-        return ResponseEntity.ok(person);
+        dataBase.deleteById(id);
+        return ResponseEntity.ok("OK"); // Erro de compatibilidade
     }
 
-    @PatchMapping(value = "/{id}")
-    public ResponseEntity<Person> Update(@PathVariable long id) {
-        log.info("The id {} was successfully updated", id);
-        var person = dataBase.find(id);
-        return ResponseEntity.ok(person);
+    @PutMapping
+    public ResponseEntity<Person> UPDATE(@PathVariable Person person) {
+        log.info("The person {} was successfully updated", person);
+        dataBase.UPDATE(person);

```

Comment:
@Danilosimoes no java utilizamos [camelcase](http://java-hunters.blogspot.com/2014/12/o-padrao-camelcase.html) para nomear metodos , variavéis... etc

Reply:
Fiz algumas mudanças no codigo, rodou no intellij normal, mas no postman não está rodando,

Reply:
@Danilosimoes o postman não roda nada, ele faz requisições HTTP coloque aqui um print do problema

Reply:
**no IntelliJ aparece esse erro aqui**
Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public org.springframework.http.ResponseEntity<br.com.people.api.domain.Person> br.com.people.api.controller.PersonController.UPDATE(br.com.people.api.domain.Person)]

E no postman aparece que não é suportado, ai eu coloco no options e ele não da nenhum erro, porém não aparece a mensagem de retorno, "OK".

Reply:
**Na função delete, essa mensagem**
: Resolved [org.springframework.web.HttpRequestMethodNotSupportedException: Request method 'DELETE' not supported]

e no postam eu coloco no options tbm, o erro sai, porém também não me retorna uma mensagem de ok!